launcher/src/backend/ethereum-services/GethService.js

```diff
@@ -7,40 +7,71 @@ export class GethService extends NodeService {
     service.setId();
     const workingDir = service.buildWorkingDir(dir);
 
-    const JWTDir = "/engine.jwt";
-    const dataDir = "/opt/data/geth";
-    const volumes = [new ServiceVolume(workingDir + "/data", dataDir), new ServiceVolume(workingDir + "/engine.jwt", JWTDir)];
+    const JWTDir = network === "devnet" ? "/execution/engine.jwt" : "/engine.jwt";
+    const dataDir = network === "devnet" ? "/execution" : "/opt/data/geth";
+    const volumes =
+      network === "devnet"
+        ? [new ServiceVolume(workingDir, dataDir), new ServiceVolume(workingDir + "/engine.jwt", JWTDir)]
+        : [new ServiceVolume(workingDir + "/data", dataDir), new ServiceVolume(workingDir + "/engine.jwt", JWTDir)];
+
+    const cmd =
+      network === "devnet"
+        ? [
+            "--http",
+            "--http.api=eth,web3,net",
+            "--http.port=8545",
+            "--http.addr=0.0.0.0",
+            "--http.corsdomain=*",
+            "--ws",
+            "--ws.api=eth,net,web3",
+            "--ws.port=8546",
+            "--ws.addr=0.0.0.0",
+            "--ws.origins=*",
+            "--authrpc.port=8551",
+            "--authrpc.vhosts=*",
+            "--authrpc.addr=0.0.0.0",
+            `--authrpc.jwtsecret=${JWTDir}`,
+            `--datadir=${dataDir}`,
+            "--allow-insecure-unlock",
+            "--nodiscover",
+            "--syncmode=full",
+            "--metrics",
+            "--metrics.expensive",
+            "--metrics.port=6060",
+            "--metrics.addr=0.0.0.0",
+          ]
+        : [
+            `--${network}`,
+            `--datadir=${dataDir}`,
+            "--state.scheme=path",
+            "--http",
+            "--http.port=8545",
+            "--http.addr=0.0.0.0",
+            "--http.vhosts=*",
+            "--http.api=eth,web3,net",
+            "--http.corsdomain=*",
```

Comment:
i think you can code this simpler. the configs are almost the same, just remove / add the un/wanted ones